extensions/wikia/ArticleComments/ArticleCommentsHooks.class.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Wikia\PageHeader\Button;
+
+class ArticleCommentsHooks {
+	public static function onAfterPageHeaderButtons( &$buttons ) {
+		$title = RequestContext::getMain()->getTitle();
+		$service = PageStatsService::newFromTitle( $title );
+		$comments = $service->getCommentsCount();
+
+		if ( $comments && ArticleCommentInit::ArticleCommentCheckTitle( $title ) ) {
+			$label =
+				wfMessage( 'article-comments-comments' )
+					->params( CommentsLikesController::formatCount( $comments ) )
+					->parse();
```

Comment:
and for params only ->escaped() should be enough (sorry for wrong hint @latata ;)